src/isInteger32.js

```diff
@@ -0,0 +1,27 @@
+import { curryN } from 'ramda';
+
+import toInteger32 from './toInteger32';
+
+/**
+ * Checks whether the passed value is a 32 bit integer.
+ *
+ * @func isFinite
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.32.0|v2.32.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link RA.isInteger32|isInteger32}
```

Comment:
```suggestion
 * @see {@link RA.toInteger32|toInteger32}
```

Reply:
Just for reference: you can directly commit my suggestion by clicking on "Commit suggestion" button. More info here: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request